lib/workers/repository/dependency-dashboard.ts

```diff
@@ -175,7 +175,7 @@ function appendRepoProblems(config: RenovateConfig, issueBody: string): string {
     );
     newIssueBody += '## Repository problems\n\n';
     newIssueBody +=
-      'These problems occurred while renovating this repository.\n\n';
+      'Renovate tried to update this repository, but found these problems.\n\n';
```

Comment:
By the way, some users search the logs for specific strings to build out their own dashboard:

- #23105

So do we consider a log message change a `patch` or `refactor` type thing then? This `refactor` may be a _breaking change_ for some of those "building my own telemetry for Renovate"-users. 🙃